### PR TITLE
Update protegrityDataTokenization README.md

### PR DESCRIPTION
**Pull request by RaphaelSanamyan**
Update ProtegrityDataTokenizationTemplate README.md
* Adding notes about avro.

### CODE REVIEW

File: v2/protegrity-data-tokenization/README.md [hunk 1]
```diff
@@ -236,6 +236,12 @@ adding support for:
 Following the template design, it is recommended that inputs are transformed into Beam Row and
 outputs are transformed from Beam Row.
 
+## Avro notes:
+
+* For the correct work of the pipeline, you must check that provided Data Schema match the data
+  that comes from Avro. Otherwise the pipelinу will fail with runtime error.
```
Comment (KhaninArtur) [inline v2/protegrity-data-tokenization/README.md]: pipeliny -> pipeline

File: v2/protegrity-data-tokenization/README.md [hunk 2]
```diff
@@ -236,6 +236,12 @@ adding support for:
...
+* The deadletters will not be saved when working with Avro.
```
Comment (KhaninArtur) [inline v2/protegrity-data-tokenization/README.md]: What do you think if we rephrase this a bit?

> All elements that fail to parse due to the wrong Avro schema won't be saved in a deadletter.

**Review by KhaninArtur — CHANGES_REQUESTED** [review]

File: v2/protegrity-data-tokenization/README.md (hunk 2 above)
Comment (RaphaelSanamyan) [reply v2/protegrity-data-tokenization/README.md]: Seems good!

**Review by KhaninArtur — APPROVED** [review]

File: v2/protegrity-data-tokenization/README.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ Supported data formats:
 
 - JSON
 - CSV
-- Avro
+- [Avro](#Avro-notes)
```
Comment (KhaninArtur) [inline v2/protegrity-data-tokenization/README.md:11]: Please, replace `#Avro-notes` with `#Avro-notes:` so the reference will be right 🙂

**Review by KhaninArtur — CHANGES_REQUESTED** [review]

File: v2/protegrity-data-tokenization/README.md (hunk 3 above)
Comment (RaphaelSanamyan) [reply v2/protegrity-data-tokenization/README.md:11]: On Github the link works correctly, but after adding ":" it doesn't work. I also checked the link in IDEA, but the link to the section doesn't work in any of these cases.

Comment (KhaninArtur) [reply v2/protegrity-data-tokenization/README.md:11]: Got you, thanks for double checking!

**Review by KhaninArtur — APPROVED** [review]

File: v2/protegrity-data-tokenization/README.md [hunk 4]
```diff
@@ -236,6 +236,12 @@ adding support for:
 Following the template design, it is recommended that inputs are transformed into Beam Row and
 outputs are transformed from Beam Row.
 
+## Avro notes:
+
+* For the correct work of the pipeline, you must check that provided Data Schema match the data
+  that comes from Avro. Otherwise the pipeline will fail with runtime error.
+* All elements that fail to parse due to the wrong Avro schema won't be saved in a deadletter.
```
Comment (KhaninArtur) [inline v2/protegrity-data-tokenization/README.md]: Could you please update it per Alex's request?

**Review by KhaninArtur — CHANGES_REQUESTED** [review]
Sorry, forgot to check if there are added recently asked changes.

**Review by KhaninArtur — APPROVED** [review]
Great, thanks! Feel free to squash and merge!

**Pull request merged**